pilot/pkg/model/config.go

```diff
@@ -573,11 +595,14 @@ func ResolveShortnameToFQDN(host string, meta ConfigMeta) string {
 // from the generic config registry
 type istioConfigStore struct {
 	ConfigStore
+
+	// AuthCache is a cache for auth(right now jwks_uri), it has some expiration logic on updating the URIs.
+	AuthCache cache.ExpiringCache
```

Comment:
The reference to authCache sound like the right place but I prefer not directly putting the implementation of authCache into istioConfigStore. It's better to have an AuthCache wrapper over cache.ExpiringCache in another file and implement the jwt url lookup and mutation function there. 

The pl

Reply:
thanks, done.